<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCorpusCommunicationDialog</class>
 <widget class="QDialog" name="AddCorpusCommunicationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QLabel" name="labelCorpusID">
     <property name="text">
      <string>Corpus ID:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="editCorpusID"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelCommunicationID">
     <property name="text">
      <string>Communication ID:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="editCommunicationID"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBoxMediaFiles">
     <property name="title">
      <string>Media File(s) to Import</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item row="0" column="0" rowspan="2">
       <widget class="QListWidget" name="listWidgetMediaFiles"/>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="commandAddMediaFile">
        <property name="text">
         <string>Add...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="commandRemoveMediaFile">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBoxAnnotationFiles">
     <property name="title">
      <string>Annotation File(s) to Import</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item row="0" column="0" rowspan="2">
       <widget class="QListWidget" name="listWidgetAnnotationFiles"/>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="commandAddAnnotationFile">
        <property name="text">
         <string>Add...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="commandRemoveAnnotationFile">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
